articles/quickstart/spa/_includes/_login_preamble.md

```diff
@@ -1,3 +1,8 @@
-The first step in adding authentication to your ${library} application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the [hosted login page](https://auth0.com/docs/hosted-pages/login).
+Use the  hosted login page to provide a way for your users to log in to your ${library} application.
 
-When authentication is successful, three items will be returned which can all be used at some point in your application: an `access_token`, an `id_token`, and the number of seconds until the `access_token` expires.
+When the user logs in, Auth0 returns three items:
+* `access_token` - an access token
```

Comment:
use this format:
        `access_token`: an access token

Also, links to the tokens docs won't be a bad idea